e2e-tests/cross-site/run

```diff
@@ -0,0 +1,99 @@
+	source_primary_endpoint=$(get_service_endpoint ${source_primary})
+
+	kubectl_bin patch pxc $source_cluster --type=merge --patch '{"spec": {"pxc":{"replicationChannels": [{"name":"source_to_replica", "isSource": true}]}}}'
+
+	kubectl_bin patch secret my-cluster-secrets -p '{"data":{"replication": "'$(echo -n new_password | base64)'"}}'
+	sleep 15
+	wait_cluster_consistency "$source_cluster" 3 2
+
+	desc 'Write data to source cluster'
+	run_mysql \
+		'CREATE DATABASE IF NOT EXISTS myApp; use myApp; CREATE TABLE IF NOT EXISTS testSourceReplica (id int PRIMARY KEY);' \
+		"-h $source_primary_endpoint -uroot -proot_password"
+	run_mysql \
+		'INSERT myApp.testSourceReplica (id) VALUES (100700)' \
+		"-h $source_primary_endpoint -uroot -proot_password"
+
+	desc "Create replica cluster"
+
+	create_infra $replica_namespace
+	spinup_pxc "$replica_cluster" "$test_dir/conf/$replica_cluster.yml"
+	sleep 60
+	replica_endpoint0=$(get_service_endpoint ${replica_cluster}-pxc-0)
+	replica_endpoint1=$(get_service_endpoint ${replica_cluster}-pxc-1)
+	replica_endpoint2=$(get_service_endpoint ${replica_cluster}-pxc-2)
+	replica_primary=$(get_proxy_primary "-h127.0.0.1 -P6032 -uproxyadmin -padmin_password" "$replica_cluster-proxysql-0")
+	replica_primary_endpoint=$(get_service_endpoint ${replica_primary})
+
+	run_mysql \
+		'DELETE FROM myApp.myApp WHERE id=100500' \
+		"-h $replica_primary_endpoint -uroot -proot_password"
+
+	kubectl_bin patch pxc $replica_cluster --type=merge --patch '{"spec": {"pxc": {"replicationChannels":[{"name": "source_to_replica", "isSource": false, "sourcesList": [{"host": "'${source_endpoint0}'", "port": 3306, "weight": 100},{"host": "'${source_endpoint1}'", "port": 3306, "weight": 100},{"host": "'${source_endpoint2}'", "port": 3306, "weight": 100}]}]}}}'
+
+	kubectl_bin patch secret my-cluster-secrets -p '{"data":{"replication": "'$(echo -n new_password | base64)'"}}'
+	sleep 15
+	wait_cluster_consistency "$replica_cluster" 3 2
+
+  desc "Check replication source-> replica works on"
+  compare_mysql_cmd "select-2" "SELECT * from myApp.testSourceReplica;" "-h $replica_endpoint0 -uroot -proot_password"
+  compare_mysql_cmd "select-2" "SELECT * from myApp.testSourceReplica;" "-h $replica_endpoint1 -uroot -proot_password"
+  compare_mysql_cmd "select-2" "SELECT * from myApp.testSourceReplica;" "-h $replica_endpoint2 -uroot -proot_password"
+
+	desc "Switch clusters over"
+	kubectl_bin config set-context $(kubectl_bin config current-context) --namespace="$namespace"
+	kubectl_bin patch pxc $source_cluster --type=merge --patch '{"spec": {"pxc": {"replicationChannels":[{"name": "source_to_replica", "isSource": false, "sourcesList": [{"host": "'${replica_endpoint0}'", "port": 3306, "weight": 100},{"host": "'${replica_endpoint1}'", "port": 3306, "weight": 100},{"host": "'${replica_endpoint2}'", "port": 3306, "weight": 100}]}]}}}'
+
+	run_mysql \
```

Comment:
Can we just switch over without deleting anything and just add new writes?

Reply:
nope. In this case we should set up replication manually with a particular point. Otherwise we try to duplicate values in source_cluster and replication becomes broken. Do you mean extract gtid and use it to start replication?

Reply:
Ok, let's just leave it as it is so that we don't overcomplicate. The goal was to check if switching the cluster from replica to source works (`isSource` option).

Reply:
yep! it works fine.